### ISSUE TITLE
Export songs to individual track files

### ISSUE THREAD

**Issue by uminokuchi** [Kermalis/VGMusicStudio#119]
It would be nice if there was an option to output all channels at once. Currently, it is troublesome to output each channel one by one.

**Comment by Kermalis** [comment]
Hello, by "channel", you mean the songs?

**Comment by uminokuchi** [comment]
Is it easier to understand the parts of the song? DS has up to 16 channels. (Since it is a Google translation, it may have been difficult to convey the nuances. I'm sorry.)

**Comment by Kermalis** [comment]
When you export the .wav, it will export all of the channels you have unmuted. So the ones you mute won't be in the .wav

**Comment by uminokuchi** [comment]
Yes. However, I think it would be easier to use if there was an option to output all channels except channel 1 with mute first, then all channels except channel 2 with mute output, then channel 3, and so on.

**Comment by Kermalis** [comment]
The top checkboxes will toggle every state:
![image](https://github.com/Kermalis/VGMusicStudio/assets/29823718/03dec644-6708-4504-9571-186cb463dba0)

So you can turn them all off at once, then turn on one at a time

**Comment by uminokuchi** [comment]
Yes. But what I'm saying is, please implement a way to do it with fewer clicks. For example, is it possible to add an item called Export songs to individual track files in the Data tab?

**Comment by Kermalis** [comment]
Yeah, that's doable. I get what you mean now. I won't be able to add it for a long time though
